test_scripts/API/VehicleData/GetVehicleData/006_GetVD_mandatory_parameters.lua

```diff
@@ -0,0 +1,60 @@
+---------------------------------------------------------------------------------------------------
+-- Description: Check that SDL processes GetVehicleData RPC with <vd_param> parameter
```

Comment:
The description and test sequence contradict each other. Please update.
And in case the script should process missing of mandatory parameters please update the script name also.